src/nodes.coffee

```diff
@@ -1414,6 +1501,9 @@ exports.Access = class Access extends Base
     else
       [@makeCode('['), name..., @makeCode(']')]
 
+  ast: ->
```

Comment:
Both `Access` and `Index` "forward" their AST generation on to a child. Again, just overriding `ast()` is as easy what I was doing previously (overriding `_toAst()` or `toAst()`)